sale_pricelist_global_rule/models/sale_order.py

```diff
@@ -0,0 +1,84 @@
+from odoo import api, fields, models
+from odoo.tools import float_compare
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    need_recompute_pricelist_global = fields.Boolean()
+    has_pricelist_global = fields.Boolean(compute="_compute_has_pricelist_global")
+
+    @api.depends("pricelist_id")
+    def _compute_has_pricelist_global(self):
+        for sale in self:
+            sale.has_pricelist_global = any(
```

Comment:
This is very limited, as the rule may not be applicable to the current lines.

Reply:
Done